app/utils/mail_utils.rb

```diff
@@ -74,10 +74,11 @@ def set_community(new_community_id, community_locales, &block)
   module_function
 
   def community_specific_sender(community)
-    if community && community.custom_email_from_address
-      community.custom_email_from_address
-    else
-      APP_CONFIG.sharetribe_mail_from_address
-    end
+    Maybe(community).id.map { |cid|
+      # TODO Use API
+      MarketplaceSenderEmail.where(community_id: cid).first
```

Comment:
Why leave a TODO here instead of implementing this part of the API and store straight away?